src/app/components/forum/topic-list/topic-list.styl

```diff
@@ -29,8 +29,10 @@
 			width: 40px
 
 		&-info
-			&-username
+			div
```

Comment:
change this to `> div` otherwise you're targeting every div inside the info, whether it's nested or not